src/option.h

```diff
@@ -1339,6 +1339,17 @@ enum
     , WV_COUNT	    // must be the last one
 };
 
+/*
+ * "indir" values for tab-local options.
+ * These need to be defined globally, so that the TV_COUNT can be used in the
```

Comment:
`TV_COUNT` isn't used anywhere

Reply:
Yes. Fix the comment or remove the definition?

Reply:
Actually I think that means the implementation is incomplete. `BV_COUNT` and `WV_COUNT` are used for arrays that store script contexts that set local options, so if 'cmdheight' needs to be a fully tab-local option it's necessary to store a script context with the tabpage as well.

Reply:
I think that should be done in a separate PR about `cmdheight`.

Reply:
Yeah, but we should still remember when this option was (re-)set right?

Reply:
I'll write another PR to make `cmdheight` as a true tab-local option.
And in the meantime I'd like to mark this PR as a WIP.